src/main/java/file/notes/App.java

```diff
@@ -18,10 +18,15 @@ public static void main(String[] args) {
             } catch (IOException e) {
                 e.printStackTrace();
             }
-            System.out.println("Success: " + command);
             if ("q".equals(command)) {
                 System.exit(0);
             }
+            else if ("create".equals(command) || "delete".equals(command) || "show".equals(command)) {
+                System.out.println("Success: " + command);
+            }
+            else {
```

Comment:
The good code-convetion is to keep else on the same line as closing brackets. Like this: } else {. The sane for else if() at the line no. 24